google-cloud-spanner/src/test/java/com/google/cloud/spanner/OpenTelemetrySpanTest.java

```diff
@@ -282,7 +266,11 @@ public void setUp() throws Exception {
             .setChannelProvider(channelProvider)
             .setOpenTelemetry(openTelemetry)
             .setCredentials(NoCredentials.getInstance())
-            .setSessionPoolOption(SessionPoolOptions.newBuilder().setMinSessions(0).build());
+            .setSessionPoolOption(
+                SessionPoolOptions.newBuilder()
+                    .setMinSessions(2)
```

Comment:
You can also have a `maxSessions` and make min=max to have a more deterministic behavior. Else if you get more than 2 requests concurrently, it will try to create more sessions. Are we making sure that current tests always utilize <= 2 sessions?

Reply:
Yes, current tests always utilize 1 session. I created 2 just for extra safe.